src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -289,20 +293,20 @@ export default class SimidMapCreative extends BaseSimidCreative {
     this.calculateRoute_();
   }
 
-  /**
+   /**
    * Creates a drop down menu where users can choose between
    * different modes of travel to display directions for.
    * @private 
    */
   createTravelChoices_() {
-    const travelChoicesContainer = document.getElementById("button_container")
+    const travelChoicesContainer = document.getElementById("adContainer")
     const travelMethod = document.createElement('select');
-    travelMethod.id = "travel_method";
+    travelMethod.id = "travelMethod";
     TRANSPORT_METHODS.forEach((transportType) =>{
       const newOption = this.createTravelOption_(transportType);
       travelMethod.add(newOption);
     });
-    travelMethod.classList.add("travel_method");
+    // travelMethod.classList.add("travel_method");
```

Comment:
we can get rid of this line!

Reply:
great, just wasn't sure why you had it initially, so was going to ask you